### PR TITLE
Added support for route prefixes.

### PR DESCRIPTION
**Pull request by sebmak**
Added the ability to use route prefixes on groups.

### CODE REVIEW

File: src/Application.php [hunk 1]
```diff
@@ -778,6 +778,10 @@ public function withEloquent()
      */
     public function group(array $attributes, Closure $callback)
     {
+        if($this->groupAttributes['prefix']){
```
Comment (GrahamCampbell) [inline src/Application.php]: cs

File: src/Application.php [hunk 2]
```diff
@@ -887,6 +891,11 @@ protected function addRoute($method, $uri, $action)
         }
 
         if (isset($this->groupAttributes)) {
+
+            if(isset($this->groupAttributes['prefix'])){
```
Comment (GrahamCampbell) [inline src/Application.php]: cs

File: src/Application.php [hunk 3]
```diff
@@ -887,6 +891,11 @@ protected function addRoute($method, $uri, $action)
         }
 
         if (isset($this->groupAttributes)) {
+
```
Comment (GrahamCampbell) [inline src/Application.php]: no bank line here please

**Comment by taylorotwell** [comment]
Implemented a little differently. Thanks.

**Comment by mikeerickson** [comment]
@taylorotwell is this merged now so it works with latest build? I composer updated this morning and still not working.

**Comment by taylorotwell** [comment]
I haven't tagged it yet. Will in the morning.

**Comment by mikeerickson** [comment]
@taylorotwell cool thx !!

**Comment by taylorotwell** [comment]
Tagged.

**Comment by mikeerickson** [comment]
@taylorotwell awesome, thx for attention on this one.  Much appreciated

**Comment by mikeerickson** [comment]
Here is where I placed files, along with screenshot of files after install

/Applications/WebStorm.app/Contents/lib/

Screenshot

http://d.pr/i/1k6mb

**Pull request closed without merging**